Description: fix common spelling errors in manual page (lintian)
Author: Jay Berkenbilt <qjb@debian.org>
Bug: http://bugzilla.maptools.org/show_bug.cgi?id=2144
Forwarded: yes

Index: tiff-3.9.4/man/TIFFReadDirectory.3tiff
===================================================================
--- tiff-3.9.4.orig/man/TIFFReadDirectory.3tiff	2010-06-18 21:30:30.143663951 -0400
+++ tiff-3.9.4/man/TIFFReadDirectory.3tiff	2010-06-18 21:30:38.643662552 -0400
@@ -119,20 +119,20 @@
 An unknown tag was encountered in the directory;
 the library ignores all such tags.
 .PP
-\fBTIFF directory is missing requred "ImageLength" field\fP.
+\fBTIFF directory is missing required "ImageLength" field\fP.
 The image violates the specification by not having a necessary field.
 There is no way for the library to recover from this error.
 .PP
-\fBTIFF directory is missing requred "PlanarConfig" field\fP.
+\fBTIFF directory is missing required "PlanarConfig" field\fP.
 The image violates the specification by not having a necessary field.
 There is no way for the library to recover from this error.
 .PP
-\fBTIFF directory is missing requred "StripOffsets" field\fP.
+\fBTIFF directory is missing required "StripOffsets" field\fP.
 The image has multiple strips, but is missing the tag that
 specifies the file offset to each strip of data.
 There is no way for the library to recover from this error.
 .PP
-\fBTIFF directory is missing requred "TileOffsets" field\fP.
+\fBTIFF directory is missing required "TileOffsets" field\fP.
 The image has multiple tiles, but is missing the tag that
 specifies the file offset to each tile of data.
 There is no way for the library to recover from this error.
Index: tiff-3.9.4/man/TIFFWriteDirectory.3tiff
===================================================================
--- tiff-3.9.4.orig/man/TIFFWriteDirectory.3tiff	2010-06-18 21:30:30.151662814 -0400
+++ tiff-3.9.4/man/TIFFWriteDirectory.3tiff	2010-06-18 21:30:38.643662552 -0400
@@ -61,7 +61,7 @@
 but instead of place it at it's old location (as 
 .IR TIFFWriteDirectory
 would) it will place them at the end of the file, correcting the pointer from
-the preceeding directory or file header to point to it's new location.  This
+the preceding directory or file header to point to it's new location.  This
 is particularly important in cases where the size of the directory and
 pointed to data has grown, so it won't fit in the space available at the
 old location.
